### PR TITLE
Update and standardise tooling

### PR DESCRIPTION
**Pull request by adamltyson**

### CODE REVIEW

**Comment by adamltyson** [comment]
Tests seem to be failing because neuromorpho.org is down.

File: morphapi/api/mpin_celldb.py [hunk 1]
```diff
@@ -137,7 +138,10 @@ def download_dataset(self):
             )
         SOURCE_DATA_DIR = "MPIN-Atlas__Kunst_et_al__neurons_all"
 
-        REMOTE_URL = "https://fishatlas.neuro.mpg.de/neurons/download/download_all_neurons_aligned"
+        REMOTE_URL = (
+            "https://fishatlas.neuro.mpg.de/neurons/download/"
+            "download_all_neurons_aligned"
```
Comment (IgorTatarnikov) [inline morphapi/api/mpin_celldb.py:143]: Is this link dead?

File: morphapi/api/mpin_celldb.py [hunk 2]
```diff
@@ -162,9 +166,11 @@ def download_dataset(self):
 
         shutil.rmtree(extracted_data_path.parent)
 
-        # # 2/1900 neurons still have a little bug, hopefully fixed in the future
+        # # 2/1900 neurons still have a little bug, hopefully
```
Comment (IgorTatarnikov) [inline morphapi/api/mpin_celldb.py:169]: Can this be removed, or is it still needed as pseudo documentation?

**Review by IgorTatarnikov — COMMENTED** [review]
Should we wait for neuromorpho.org to come back up?

**Comment by adamltyson** [comment]
[neuromorpho is down until at least january](https://neuromorpho.org/). @IgorTatarnikov haven't made any functional changes in this PR, so I suggest we raise issues and merge as is?

**Comment by adamltyson** [comment]
I've raised #47, #48 & #49

**Comment by IgorTatarnikov** [comment]
Sounds good 👍, thanks!

**Review by IgorTatarnikov — APPROVED** [review]

**Pull request merged**